src/Illuminate/Database/Query/Builder.php

```diff
@@ -2716,7 +2717,7 @@ public function insert(array $values)
             foreach ($values as $key => $value) {
                 ksort($value);
 
-                $values[$key] = $value;
+                $values[$key] = array_merge($value, $this->resolveInsertDefalts($defaultValues));
```

Comment:
`array_merge` in a for loop is not that good, also should not this be the other way around? The defaults will overwrite the values of the first parameter.